CONTRIBUTING.md

```diff
@@ -36,8 +36,8 @@ We use the famous [gitflow](http://nvie.com/posts/a-successful-git-branching-mod
 
 # Discussion
 
-- Facebook group (for Thai NLP Discussion only): https://www.facebook.com/groups/thainlp 
```

Comment:
- [ ] Add Discussion

Reply:
That was my mistake, I already fixed it. 😥